scripts/asm-installer/install_asm

```diff
@@ -2352,12 +2352,7 @@ install_managed_components() {
   kubectl apply -f "${BASE_REL_PATH}"
   kubectl replace -f "${MANAGED_WEBHOOKS}"
 
-  PARAMS="-f ${MANAGED_MANIFEST}"
-  PARAMS="${PARAMS} -c ${KUBECONFIG}"
-  PARAMS="${PARAMS} --skip-confirmation"
-  PARAMS="${PARAMS} --set revision=${REVISION_LABEL}"
-
-  info "Installing ASM managed control plane components..."
+  info "Configuring ASM managed control plane components..."
```

Comment:
That doesn't fix the problem - istioctl install is still there.

Also: it would be great to print "To install gateway, run .... "

Reply:
Removed the install. If we're going to print anything I'd rather print a link to the docs like the other examples in the outro.

Reply:
That's fine too.